src/main.css

```diff
@@ -0,0 +1,43 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    font-family: Helvetica;
+    background-color: lightgray
+};
+
+nav {
+    margin: 10px 10px;
+    font-size:;
+}
+
+h1 {
+    font-family: 'DM Serif Text', serif;;
+    text-align: center;
+    margin: 50px;
+}
+
+h2, h3, nav {
+    font-family: 'DM Serif Text', serif;
+    margin: 30px 20px;
+}
+
+p {
+    margin: 10px;
+}
+
+ol, ul, li {
+    margin: 0px;
+}
+
+img{margin:20px;}
```

Comment:
Open these curlys up too! If you want your imgs to have a similar height for your lis, I suggest giving them a height or width. Also, a more detailed selector would be best (#instruments li img) and that would probably be in your products.css